tools/tsp-client/src/git.ts

```diff
@@ -0,0 +1,82 @@
+import { execSync, spawn } from "child_process";
+
+export function getRepoRoot(): string {
+    return execSync('git rev-parse --show-toplevel').toString().trim();
```

Comment:
use `exec` instead of `execSync` no sense in hanging the process while git does its thing

Reply:
Tracking issue for future PR: https://github.com/Azure/azure-sdk-tools/issues/7045